test/CMakeLists.txt

```diff
@@ -111,8 +112,11 @@ add_custom_command(
         ${CMAKE_CURRENT_SOURCE_DIR}/test_axpy_8bit.pbtxt
         ${CMAKE_CURRENT_BINARY_DIR}/test_axpy_8bit.pbtxt
     COMMAND ${CMAKE_COMMAND} -E create_symlink
-        ${CMAKE_CURRENT_SOURCE_DIR}/test_graph_1.pbtxt
-        ${CMAKE_CURRENT_BINARY_DIR}/test_graph_1.pbtxt
+        ${CMAKE_CURRENT_SOURCE_DIR}/test_mixed_tensors_pipelined.pbtxt
+        ${CMAKE_CURRENT_BINARY_DIR}/test_mixed_tensors_pipelined.pbtxt
```

Comment:
not sure about this name, since the graph has nothing to do with mixed tensors. It can be used for any test, it is just a bunch of args, operations (add, mul) and retvals

Reply:
The file had a very generic name which per chance I used while dumping graphs and then this existing file was overwritten. How about test_general_graph.pbtxt, @shresthamalik ?